WordPress/Classes/ViewRelated/Post/EditPostViewController.swift

```diff
@@ -2,6 +2,12 @@ import UIKit
 
 class EditPostViewController: UIViewController {
 
+    enum editor: String {
```

Comment:
@catehstn We could put those strings into `WPAnalytics.h` but in reality it should probably be another enum mapping since each Tracker could have different formats for property names and possibly value formats.